core/src/main/java/com/oath/oak/OakMapBuilder.java

```diff
@@ -47,66 +47,60 @@ public OakMapBuilder() {
     this.memoryAllocator = null;
   }
 
-  public OakMapBuilder setKeySerializer(OakSerializer<K> keySerializer) {
+  public OakMapBuilder<K, V> setKeySerializer(OakSerializer<K> keySerializer) {
```

Comment:
These <K,V> supplements are not really part of this pull request, correct?

Reply:
Correct.
Im piggybacking this pr to fix some coding errors.
I also added jmh benchmarks and that needed some reorg of the code into modules.
I also fixed some test errors that caused tests to fail because autoclosable objects where not in try-with-resource block.